### PR TITLE
Refactored forms and added image field in student registration form.

### PR DESCRIPTION
**Pull request by KD-3**
### Description
Refactored Company and student registration forms and added image field in student registration form.
Modified accounts/forms.py, accounts/models.py, accounts/views.py, student/register.html, company/register.html

### Type of Change:
- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by maintainers)

### Checklist:

- [x] My PR follows the style guidelines of this project

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/accounts/forms.py [hunk 1]
```diff
@@ -28,25 +28,28 @@ class StudentRegisterForm(UserCreationForm):
...
-    nationality = forms.ChoiceField(choices=StudentProfile.NATION)
-    permanent_address = forms.CharField(widget=forms.Textarea)
-    current_address = forms.CharField(widget=forms.Textarea)
-    x_year = forms.IntegerField(max_value=2050, min_value=1980)
-    x_board_name = forms.CharField(max_length=100)
-    x_percentage = forms.CharField(max_length=10)
-    xii_year = forms.IntegerField(max_value=2050, min_value=1980)
-    xii_board_name = forms.CharField(max_length=100)
-    xii_percentage = forms.CharField(max_length=16)
+    username = forms.CharField(max_length=11, help_text="Enter your Roll number, this will be used to login",
```
Comment (anshulahuja98) [inline src/accounts/forms.py]: Where is the regex?

File: src/accounts/models.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ class StudentProfile(models.Model):
     xii_percentage = models.CharField(max_length=10)
     banned = models.BooleanField(default=False)
     placed = models.BooleanField(default=False)
+    std_image = models.ImageField(default=True, upload_to='student_images')
```
Comment (anshulahuja98) [inline src/accounts/models.py]: `default=True` in an image field !?

File: src/accounts/models.py [hunk 3]
```diff
@@ -67,7 +68,7 @@ class CompanyProfile(models.Model):
                                         through_fields=('company', 'student'), related_name='joboffers')
     internship_offers = models.ManyToManyField(StudentProfile, through='company.InternshipOffer',
                                                through_fields=('company', 'student'), related_name='internshipoffers')
-    contact = models.CharField(max_length=20)
+    contact = models.CharField(max_length=10)
```
Comment (anshulahuja98) [inline src/accounts/models.py]: why did you do reduce this

File: src/accounts/views.py [hunk 4]
```diff
@@ -33,6 +34,21 @@ def form_valid(self, form):
         StudentRegisterFormView.create_profile(user, **form.cleaned_data)
         return super(StudentRegisterFormView, self).form_valid(form)
 
+    def std_image_view(self, request):
```
Comment (anshulahuja98) [inline src/accounts/views.py]: Why is this needed?

File: src/spc_django/settings.py [hunk 5]
```diff
@@ -43,7 +43,7 @@
     'student.apps.StudentConfig',
     'accounts.apps.AccountsConfig',
     'company.apps.CompanyConfig',
-    'main.apps.MainConfig'
+    'main.apps.MainConfig',
```
Comment (anshulahuja98) [inline src/spc_django/settings.py]: Why?

File: src/accounts/models.py (hunk 3 above)
Comment (geetesh-gupta) [reply src/accounts/models.py]: Provide a one-off default for the field.
